tests/virt_autotest/login_console.pm

```diff
@@ -120,6 +120,12 @@ sub login_to_console {
             #send key 'up' to stop grub timer counting down, to be more robust to select xen
             send_key 'up';
             save_screenshot;
+            #If sol console is stuck, reconnect it and resend key up
+            if (check_screen 'virttest-bootmenu-xen-kernel-count-down', 1) {
```

Comment:
I believe this is an effective way to proceed with xen grub menuentry selection when sol is not functioning well. Good job! 

Just a minor comment, the timeout is 1 second(so short)? Can it survive most such situations?

Reply:
In my test, it is enough and can survive.
also add it to 2 seconds, it's much enough.
It must be completed in count-down default 10 seconds.